### PR TITLE
Doc update - Integrations and Migration-basics

### PR DESCRIPTION
**Pull request by FelipeAdachi**
- removed integration section and files
- updated v0 to v1 migration - basics documentation

## Description

This repository needs 

The commits in this pull request will 

## Changes

- Imperative commit or change title (commit ID) 
  - Add some bullet points to explain the change
- Next commit or change (commit ID)

## Related

Relates to organization/repo#number

Closes organization/repo#number

- [ ] I have reviewed the [Guidelines for Contributing](CONTRIBUTING.md) and the [Code of Conduct](CODE_OF_CONDUCT.md).

### CODE REVIEW

File: python/docs/migration/basics.md [hunk 1]
```diff
@@ -1,52 +1,43 @@
-# v0 to v1 Basics
+# Basics - v0 to v1
 
-## Performance Improvement
```
Comment (FelipeAdachi) [inline python/docs/migration/basics.md:3]: Someone with more context on the specifics of how performance was improved could provide me with a simple description, and I can update.

File: python/docs/migration/basics.md [hunk 2]
```diff
@@ -1,52 +1,43 @@
...
-- We prioritized columnar operations
 
-### Microbatching by Default
+Based on user feedback, the following areas of improvement have been the focus for whylogs v1.
 
-- Single operations are batched into micro batches
+### Performance Improvements
 
-## Extensible APIs
+Users will see substantial improvements to performance, allowing larger datasets to be profiled in much less time.
```
Comment (FelipeAdachi) [inline python/docs/migration/basics.md]: Someone with more context on the specifics of how performance was improved could provide me with a simple description, and I can update.

File: python/docs/migration/basics.md [hunk 3]
```diff
@@ -1,52 +1,43 @@
-# v0 to v1 Basics
+# Basics - v0 to v1
 
-## Performance Improvement
+whylogs v1 has been released on **May 31st, 2022**. While this release represents an exciting milestone for the whyogs library, there are some important implications for current users of whylogs v0. Most notably, whylogs v1 comes with a simplified and more intuitive API. This means that if you choose to upgrade to v1, code changes will be required.
 
-* Native integration with
+Users can visit this [migration guide](https://docs.whylabs.ai/docs/v1-migration/) to assist with migrating their code to whylogs v1.
 
-### Columnar Operations
```
Comment (FelipeAdachi) [inline python/docs/migration/basics.md:7]: I removed the sections about Columnar Operations for now. If someone could provide me with one or two phrases about this, I can update it.

Comment (murilommen) [reply python/docs/migration/basics.md:7]: from the v1 launch post: One of the major causes for performance improvements with whylogs v1 is a change from row-level operations to columnar operations. Columnar operations allow us to take advantage of [vectorization](https://www.pythonlikeyoumeanit.com/Module3_IntroducingNumpy/VectorizedOperations.html#Vectorized-Operations) built into the NumPy and pandas packages, which significantly speeds up the process of generating profiles by pushing summarization from slow Python code to lightning-fast C code. Importantly, we can take advantage of vectorization without changes to the end user experience.

read more at: https://whylabs.ai/blog/posts/data-logging-with-whylogs

File: python/docs/migration/basics.md (hunk 2 above)
Comment (murilommen) [reply python/docs/migration/basics.md]: i think you could also use what was used in the v1 post

Comment (FelipeAdachi) [reply python/docs/migration/basics.md]: Thanks Murilo!
resolved

**Review by murilommen — APPROVED** [review]
lgtm

**Pull request merged**